src/elements/content-uploader/ContentUploader.js

```diff
@@ -768,6 +768,7 @@ class ContentUploader extends Component<Props, State> {
         item.api = this.getUploadAPI(file, options);
         item.progress = 0;
         item.status = STATUS_PENDING;
+        delete item.error;
```

Comment:
Should we have a unit test to go along with this to avoid a regression in the future?